### PR TITLE
PLAT-407: forbid tolerable + validated isolation

### PR DESCRIPTION
**Pull request by sexyhamster**
Forbid tolerable + validated isolation for now.

### CODE REVIEW

File: ledger-core/virtual/integration/method_test.go [hunk 1]
```diff
@@ -1534,3 +1535,179 @@ func TestVirtual_Method_ForObjectWithMissingState(t *testing.T) {
...
+					objectRef,
+					execute.NewStateID(pn, []byte("ok case")),
+					classRef,
+					[]byte("ok case"),
+				)
+			},
+			callResult: []byte("ok case"),
+		},
+		{
+			name:         "Method tolerable + dirty cannot be executed",
```
Comment (MikhailZhukov) [inline ledger-core/virtual/integration/method_test.go:1573]: misprint dirty->validation?

File: ledger-core/virtual/integration/method_test.go [hunk 2]
```diff
@@ -1534,3 +1535,179 @@ func TestVirtual_Method_ForObjectWithMissingState(t *testing.T) {
...
+						AddStart(nil, &execution.Update{
+							Type:   execution.Done,
+							Result: result,
+						})
+
+				}
+
+				server.SendPayload(ctx, &pl)
+			}
+
```
Comment (MikhailZhukov) [inline ledger-core/virtual/integration/method_test.go:1700]: maybe you need to change pulse and check in VStateReport that object state was(not) changed?

File: ledger-core/virtual/integration/method_test.go (hunk 1 above)
Comment (sexyhamster) [reply ledger-core/virtual/integration/method_test.go]: fixed

File: ledger-core/virtual/integration/method_test.go (hunk 2 above)
Comment (sexyhamster) [reply ledger-core/virtual/integration/method_test.go:1700]: don't need to. I'm checking that runner was never called.

File: ledger-core/virtual/execute/execute.go [hunk 3]
```diff
@@ -333,6 +336,14 @@ func negotiateIsolation(methodIsolation, callIsolation contract.MethodIsolation)
 	return res, nil
 }
 
+func checkAllowedIsolation(isolation contract.MethodIsolation) {
```
Comment (bigbes) [inline ledger-core/virtual/execute/execute.go:339]: I don't see a single reason why you've extracted that check to separate function, but it's not important.

Comment (sexyhamster) [reply ledger-core/virtual/execute/execute.go]: inlined the function

**Pull request merged**